src/main/scala/mesosphere/marathon/MarathonScheduler.scala

```diff
@@ -136,7 +137,7 @@ class MarathonScheduler @Inject() (
   protected def suicide(removeFrameworkId: Boolean): Unit = {
     log.error(s"Committing suicide!")
 
-    if (removeFrameworkId) Await.ready(frameworkIdUtil.expunge(), config.zkTimeoutDuration)
+    if (removeFrameworkId) Await.ready(frameworkIdRepository.delete(), config.zkTimeoutDuration)
```

Comment:
Don't think anything changed there... the name of `expunge` is just `delete` now.